### ISSUE TITLE
array size greater than 21, received as json.

### ISSUE THREAD

**Issue by Saivenkatesh** [expressjs/multer#138]
If I have an array of fields with names including the index. Say my fields are label[0],label[1].. upto label[20], it works fine. But when I include label[21] in the request, instead of receiving it as an array, I observe that req.body.label is a json object with keys '0','1','2'...'21'. 
```
----WebKitFormBoundary7MA4YWxkTrZu0gW
Content-Disposition: form-data; name="userId"

userid0
----WebKitFormBoundary7MA4YWxkTrZu0gW
Content-Disposition: form-data; name="label[0]"

1
----WebKitFormBoundary7MA4YWxkTrZu0gW
Content-Disposition: form-data; name="label[1]"
...
```
However, when I donot provide the index in the form and give it as label[], it works fine. Why is this happening? 

```
----WebKitFormBoundary7MA4YWxkTrZu0gW
Content-Disposition: form-data; name="userId"

userid0
----WebKitFormBoundary7MA4YWxkTrZu0gW
Content-Disposition: form-data; name="label[]"

1
----WebKitFormBoundary7MA4YWxkTrZu0gW
Content-Disposition: form-data; name="label[]"
...
```

**Comment by LinusU** [comment]
This is a limitation in `qs` and at the moment you can't change it without changing multer.

If you want to you can try out the code in #131 which dosen't use `qs`, hopefully we'll merge that and release a new release soon.

**Comment by LinusU** [comment]
This is now fixed in multer `1.0.0`
